agent_runtime_profile/CLAUDE.drama.md

```
@@ -78,12 +78,12 @@ agent session 的当前工作目录（cwd）已绑定到当前项目根，**所
 - `lib/` - 共享 Python 库（多供应商图像 / 视频 / 文本生成抽象层、项目管理）
 - `agent_runtime_profile/.claude/skills/` - 可用的 skills
 
-## 架构：编排 Skill + 聚焦子任务
+## 架构：编排 Skill + 聚焦子智能体
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)